symbols/arm9.yml

```
@@ -10501,9 +10511,29 @@ arm9:
       description: |-
         Validates a legendary challenge letter mission by checking whether its dungeon matches the restricted dungeon of some mission_rescue_bin struct (alongside some other conditions).
         
-        r0: mission_rescue_bin struct pointer
+        r0: mission_template struct pointer
         r1: mission struct pointer
         return: bool
+    - name: AppendMissionTitle
+      address:
+        EU: 0x2060D8C
+      description: |-
+        Generates the mission title of a mission, and appends it to the string location.
```

Minor formatting, put a blank line between the main description and the param list:

```suggestion
        Generates the mission title of a mission, and appends it to the string location.
        
```